### PR TITLE
Add `area-chart` icon

### PR DESCRIPTION
**Pull request by nstokoe**

### CODE REVIEW

**Review by ericfennis — COMMENTED** [review]
@nstokoe Welcome to the Lucide project!
Nice icon! We try to have 2px between the lines.
Can you try to move up the top line, so it will follow our [guidelines](https://lucide.dev/guide/design/icon-design-guide).
image

**Comment by nstokoe** [comment]
@ericfennis Thanks for the quick review and kind welcome. 

I had the same thought after I submitted the PR and saw the preview at 2px stroke weight. Sorry for missing that detail in the guidelines. 

It's now fixed in https://github.com/lucide-icons/lucide/commit/464e15b58858ad2b657759664fbf76738c72241f.

File: icons/area-chart.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 3V21H21" />
+  <path d="M19 8L14 13L10 9L7 12" />
+  <path d="M7 12V17" />
+  <path d="M19 17H7" />
+  <path d="M19 8V17" />
```
Comment (karsa-mistmere) [inline icons/area-chart.svg:16]: Nice icon!

Code-wise the area could be an actual area:
```suggestion
  <path d="M3 3V21H21" />
  <path d="M7 12V17H19V8L14 13L10 9L7 12Z" />
```

File: icons/area-chart.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 3V21H21" />
+  <path d="M7 12V17H19V8L14 13L10 9L7 12Z" />
```
Comment (jguddas) [inline icons/area-chart.svg:13]: This path can be optimized.

https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit?PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMyAzdjE4aDE4IiAvPgogIDxwYXRoIGQ9Ik03IDEydjVoMTJWOGwtNSA1LTQtNFoiIC8+Cjwvc3ZnPg==

**Review by jguddas — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]
@nstokoe Nicee work! LGTM!

**Pull request merged**